src/I18n/Formatter/IcuFormatter.php

```diff
@@ -51,7 +51,7 @@ public function format($locale, $message, array $vars)
             return $this->_formatMessage($locale, $message, $vars);
         }
 
-        if (isset($vars['_context']) && isset($message['_context'])) {
+        if (isset($vars['_context'], $message['_context'])) {
```

Comment:
> If multiple parameters are supplied then isset() will return TRUE only if all of the parameters are set

The behavior of isset is unambiguous.

I'm don't care too much, but it does lead to shorter lines of code.